crates/virtio-queue/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "virtio-queue"
+version = "0.1.0"
+authors = ["The Chromium OS Authors"]
+description = "virtio queue implementation"
+repository = "https://github.com/rust-vmm/vm-virtio"
+keywords = ["virtio"]
+readme = "README.md"
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+
+[features]
+test-utils = []
+
+[dependencies]
+vm-memory = ">=0.4.0"
+vmm-sys-util = ">=0.4.0"
```

Comment:
nit: Let's use the same `vmm-sys-util = ">=0.8.0"` as in the other crates.

Reply:
Done!